integrations-and-supported-tools/sagemaker/custom-docker-container/Using-Neptune-with-custom-container.ipynb

```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "abc6583e-be23-457f-9f35-d5e790aa9636",
+   "metadata": {},
+   "source": [
+    "# Using Neptune together with Amazon SageMaker training jobs \n",
+    "\n",
+    "<div class=\"alert alert-info\">You can run this part of the notebook either locally or from AWS SageMaker Notebook. It would need additional dependencies such as AWS CLI tools and Docker.</div>\n",
```

Comment:
```suggestion
    "<div class=\"alert alert-info\">You can run this part of the notebook either locally or from a SageMaker notebook. It would need additional dependencies such as AWS CLI tools and Docker.</div>\n",
```